CONTRIBUTING.md

```diff
@@ -1,16 +1,29 @@
 # Contributing Guide
 
-To contribute you will need to ensure you have the following setup:
+## Setting up a development environment
 
-- You have a working [Docker environment](https://docs.docker.com/engine).
-- You have a working [Ruby environment](https://ruby-doc.org).
+### Using nix
 
-Then clone the repo and run ci/build.sh for building and running unit test.
+1. [Install nix](https://nixos.org/download.html)
 
-```text
-git clone https://github.com/SumoLogic/sumologic-kubernetes-collection.git
-./ci/build.sh
-```
+   ```
+   sh <(curl -L https://nixos.org/nix/install) --daemon
+   ```
+
+1. Run `nix-shell` in the project root and wait for dependencies to be installed
+1. In order to run integration tests, you'll need the Docker daemon installed separately
```

Comment:
Not sure what this means. Can you provide an example?

Reply:
What is unclear about it? Integration tests require Docker, and Nix can't install the daemon part of it, so the user needs to do it themselves.